test/Java8andUp/testacc.policy

```diff
@@ -0,0 +1,26 @@
+//  Copyright (c) 2016, 2018 IBM Corp. and others
```

Comment:
The start year should not be 2016 for a new file.

Reply:
Fixed.